### PR TITLE
adds Asset.getContract

### PR DESCRIPTION
**Pull request by ElmoArmy**
### Description of changes proposed in this pull request:

- ```getContract``` Function to expose private variable on the Asset class. This will be useful to frontend users. 
- eslint warning fix, removes unused parameter ```pairName```.

### Checks:

- [X] Have you followed the guidelines in our Contributing document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### New Feature Submission:

- [X] Have you successfully ran tests with your changes locally?
- [X] Have you lint your code locally prior to submission?

### Changes to Existing Features:

- [X] Have you added an explanation of what your changes do and why you'd like us to include them?
- [X] Have you written new tests for your core changes, if applicable?

### CODE REVIEW

File: src/lib/Asset.ts [hunk 1]
```diff
@@ -297,6 +297,13 @@ class Asset {
     }
   }
 
+  /**
+   @notice Gets the contract
```
Comment (ElmoArmy) [inline src/lib/Asset.ts:301]: @byterose How do I write the docs for this? I was thinking 
```suggestion
   @notice Gets the contract
   @returns Contract as an ExpiringMultiPartyEthers or LongShortPairEthers.
```

That seem descriptive enough?

Comment (byterose) [reply src/lib/Asset.ts:301]: I would suggest to write it like this
```   
@notice Gets the contract
@returns Contract of type ExpiringMultiPartyEthers or LongShortPairEthers.
```

File: src/lib/Asset.ts [hunk 2]
```diff
@@ -297,6 +297,14 @@ class Asset {
...
+   @returns Contract of type ExpiringMultiPartyEthers or LongShortPairEthers.
+   */
```
Comment (byterose) [inline src/lib/Asset.ts:303]: I think you should add the `*` sign like this.
```
/**
 * @notice These stars remind me of what's above us. 
 */
```

**Review by byterose — CHANGES_REQUESTED** [review]

**Comment by byterose** [comment]
Ups...
Sorry cli mistake

**Pull request merged**